Vault-Deployment/values/homer.yaml

```diff
@@ -157,13 +157,13 @@ configmap:
             # logo: "path/to/logo"
             items:
               - name: "Vault UI"
-                logo: "https://vault-ui.playground.lab/ui/favicon-c02e22ca67f83a0fb6f2fd265074910a.png"
+                logo: "https://www.svgrepo.com/show/448679/vault.svg"
```

Comment:
@Xelef2000 my icon was missing :smile_cat:

Reply:
Did you check why it was missing? Could be because the TLS certificate didn't have a exclution in your browser yet. I don't like that it would be fetched from a weird 3rd party. Otherwise you could use https://raw.githubusercontent.com/hashicorp/vault/main/ui/public/vault-logo.svg